lib/ret/hub.ex

```diff
@@ -701,9 +807,30 @@ end
 
 # Permissions for un-authenticated clients
```

Comment:
Probably should update this comment to mention app tokens.

Reply:
Done. 
Speaking of this -- I noticed a few things that slowed me down while using `Canada.Can` that I'll share as food for thought:
- I want to completely separate `:reticulum_app_token` from other `Atom`s, but don't know how I could have done that. I'm tempted to create a new type to stand in for this atom just so I can isolate its permissions.
- I want to return values besides `true` or `false` to indicate what failed so that we can return good error messages to users. e.g. Differentiate between `"Token is missing required scope :write_rooms"` and `"The account associated with this token have permission to update this hub."` 
- It does not seem to me like the syntactic sugar (provided by the macro) that changes 
```elixir
can?(subject, action(argument))
``` 
into 
```elixir 
Canada.Can.can?(subject, :action, argument)
```
helps me, and it confused me for a while before I checked the source. When I was first learning elixir, I assumed I was not finding the function definitions for some reason. It's possible that this is a kind of "rookie mistake" that can't be helped. I still prefer the "raw" version and would like to go back and remove use of the macro. (The macro is what inspired me to name this function `do_create_room` instead of `create_room` so that I didn't confuse a potential reader and have them think I'm calling the same function twice. https://github.com/mozilla/reticulum/blob/feature/graphql-2/lib/ret/api/rooms.ex#L75)

Reply:
Good points. I don't have clear answers to how we could be using Canada better (or perhaps taking a different approach altogether), but I think the implementation you've got so far is pretty good.